### PR TITLE
Added clarifications for new users.

### PR DESCRIPTION
**Pull request by hickey**
## Description

Added text to assist users not familiar with Kubernetes or how the networking plugins should work. In most cases it is just clarifications of where to find the items being discussed.

## Todos

- [ ] Tests
- [X] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by hickey** [comment]
I can sign the first CLA, but the second one just displays the CLA with no buttons to accept the CLA. :-(

**Comment by caseydavenport** [comment]
I think you're good to go on the CLA.

File: v2.4/getting-started/kubernetes/installation/hosted/kubeadm/index.md [hunk 1]
```diff
@@ -4,7 +4,11 @@ redirect_from: latest/getting-started/kubernetes/installation/hosted/kubeadm/ind
 ---
 
 This document outlines how to install Calico, as well as a as single node
-etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm.
+etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm.
```
Comment (tmjd) [inline v2.4/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: It looks like several lines have white space at the end of the line, like this line.  Could you clean up any white space at the end of the lines you've changed or added.

File: v2.4/getting-started/kubernetes/installation/hosted/kubeadm/index.md [hunk 2]
```diff
@@ -4,7 +4,11 @@ redirect_from: latest/getting-started/kubernetes/installation/hosted/kubeadm/ind
 ---
 
 This document outlines how to install Calico, as well as a as single node
-etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm.
+etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm. 
+If you have already build your cluster with kubeadm, please review the
```
Comment (tmjd) [inline v2.4/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: nit: build -> built

File: v2.4/getting-started/kubernetes/installation/hosted/kubeadm/index.md (hunk 1 above)
Comment (hickey) [inline v2.4/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: I am not seeing any lines with spaces at the end of lines. I even cloned the repo and edited the file and could not find a lines with spaces at the end.

Comment (tmjd) [inline v2.4/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: I created a PR against your branch to show the spaces at the end of the lines.  https://github.com/hickey/calico/pull/1
Feel free just to update your commit with the changes.

**Comment by tmjd** [comment]
Also before we merge this we'll want the same changes made to the `master` directory as that is where we cut new doc releases from.

File: v2.4/getting-started/kubernetes/installation/hosted/kubeadm/index.md (hunk 1 above)
Comment (hickey) [inline v2.4/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: Pardon my stupidity. I was in the middle of a meeting while I was look at this and did not bother to think about checking out the branch with my changes. Fixed now.

File: v2.4/getting-started/kubernetes/installation/hosted/kubeadm/index.md (hunk 2 above)
Comment (hickey) [inline v2.4/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: Fixed.

**Comment by tmjd** [comment]
Looks good. Please apply your changes to the 'master' directory and then squash all your changes into one commit.

File: master/getting-started/kubernetes/installation/hosted/kubeadm/index.md [hunk 3]
```diff
@@ -1,9 +1,14 @@
 ---
 title: Kubeadm Hosted Install
+redirect_from: latest/getting-started/kubernetes/installation/hosted/kubeadm/index
```
Comment (tmjd) [inline master/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: We don't want to add this line to the master docs version.  Please remove it, thanks.

Comment (hickey) [inline master/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: Sorry, should have looked at the master file before copying over to it.... Just updated again.

**Comment by tmjd** [comment]
Going to close and re-open this to see if that fixes the CLA check.

**Comment by tmjd** [comment]
@hickey I think it looks like the email address on the commit is not the email address associated with your github account.  Could you check on that? It looks like Gerald.Hickey@smartsheet.com is on the commit.  Is that associated with your github account?

**Comment by hickey** [comment]
Ah, no it would not be added to the account. Let me amend the commit and see that will let it pass.

**Comment by hickey** [comment]
That should be better. Looks like the CLA check is passing now.

**Comment by hickey** [comment]
Sorry, that was probably me last night. It was very late, trying to get a bunch of things cleared off my plate and I was not thinking when I saw the PR and just merged it. Should be back to a single commit for the two index.md files for the docs. And just fixed the author on the commit again. Hopefully we are good to go now. I will keep my hands off of it! :-)

**Comment by tmjd** [comment]
@ozdanborne Want to take a peek at this then merge it if you're good with it?
@hickey Thanks again for the contribution.

**Comment by ozdanborne** [comment]
PR 1000! Woohoo!

**Pull request merged**